drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/question_twenty_four.rb

```diff
@@ -29,7 +29,10 @@ class QuestionTwentyFour < Base
 
           answer = @report.answer(question: table_name, cell: cell)
 
-          members = universe.members.where(population_clause).where(assessment_clause)
+          members = universe.members.
+            where(population_clause).
+            where(assessment_clause).
+            where(a_t[:project_type].eq(12))
```

Comment:
Do we want a comment here?  (only valid for homeless prevention) seems like it might be useful in the future.